src/xrGame/ui/UIPdaWnd.cpp

```diff
@@ -125,6 +125,28 @@ void CUIPdaWnd::Init()
     UITabControl->SetAutoDelete(true);
     AttachChild(UITabControl);
     CUIXmlInit::InitTabControl(uiXml, "tab", 0, UITabControl);
+
+    const std::tuple<shared_str, shared_str> socIds[] = 
+    {
+        {"eptMap", "1"},
+        {"eptDiary", "2"},
+        {"eptContacts", "3"},
+        {"eptStatistics", "5"},
+        {"eptEncyclopedia", "6"}
+    };
+
+    for (u32 i = 0, len = UITabControl->GetTabsCount(); i < len; i++)
+    {
+        CUITabButton* btn = UITabControl->GetButtonByIndex(i);
+        if (!btn) continue;
+
+        for (const auto& [replace, id] : socIds)
+        {
+            if (btn->m_btn_id != id) continue;
+            btn->m_btn_id = replace;
```

Comment:
Всегда переносим на следующую строку
if отдельно, continue отдельно.
В итоге, получается так:
```suggestion
                if (btn->m_btn_id == id)
                    btn->m_btn_id = replace;
```